project-base/app/config/domains_urls.yml.dist

```diff
@@ -1,6 +1,6 @@
 domains_urls:
     -   id: 1
-        url: http://127.0.0.1:8000
```

Comment:
This is awesome, but also it needs an update of documentation. I saw mentions in ./docs//installation/native-installation.md, ./docs//installation/installation-using-docker-application-setup.md, maybe delete mentions of second network interface in /docs//installation/installation-using-docker-macos.md.